CHANGELOG.rst

```diff
@@ -14,16 +14,25 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+
+[3.1.0] - 2023-07-21
+~~~~~~~~~~~~~~~~~~~~
+
+Added
++++++
+* Added Django42 support in CI.
+* Removed old versions of celery and drf version from ci.
+
 Removed
 +++++++
-
 * Removed Python 3.5 support.
 
 Chore
 +++++
 * Switch from ``edx-sphinx-theme`` to ``sphinx-book-theme`` since the former is
   deprecated
 * Upgraded celery to latest 5.x version.
+* Add Support for Django 4.2
```

Comment:
This line already exists in Added section of changelog.

Reply:
oky, removed from Chore